Design rules/readme.md

```diff
@@ -104,40 +104,181 @@ _**Ratio:**_ Consistentie in Casing van namen is voor zowel developers als desig
 Datum opname : 17-02-2021
 Datum wijziging : 17-02-2021
```

Comment:
Je hebt hier de 'Datum opname' en 'Datum wijziging' opgenomen. Is er een reden waarom je dat niet consequent ook bij de andere DR's doet (i.i.g. wat betreft 'Datum opname')?

Reply:
Geen reden, alsnog opgenomen